osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -116,7 +116,7 @@ public CatcherAnimationState CurrentState
         /// <summary>
         /// Width of the area that can be used to attempt catches during gameplay.
         /// </summary>
-        public readonly float CatchWidth;
+        public float CatchWidth;
```

Comment:
This is now publically mutable now and it probably should not be. Make this `{ get; private set; }`.